x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/generate_latest_transform.ts

```diff
@@ -79,3 +71,18 @@ export function generateLatestTransform(
     },
   };
 }
+
+function generateLatestIdentityFieldsAggregations(definition: EntityDefinition) {
+  return definition.identityFields.reduce(
+    (aggs, identityField) => ({
+      ...aggs,
+      [`entity.identityFields.${identityField.field}`]: {
+        terms: {
+          field: identityField.field,
+          size: 10,
```

Comment:
strictly, we only need 1 here, right?

Reply:
Yes, I just wonder if it's worth being lenient to catch possible edge cases?